number_theory/numeric.h

```diff
@@ -43,6 +45,54 @@ std::pair<T, T> exgcd(T a, T b) {
   return std::make_pair(xa, ya);
 }
 
+// Accumulates values according to the binary representation of |binary|.
```

Comment:
I think we should move this to unnamed namespace as a private utility function. If it will be used by other header files, then we can create `utility.h` and define this function there as a public utility function.

Reply:
moved to utility.h